chains/mainnet/kiichain.json

```
@@ -0,0 +1,31 @@
+{
+  "chain_name": "kiichain",
+  "api": [
+    {
+      "provider": "KiiChain",
+      "address": "https://lcd.kiivalidator.com"
+    }
+  ],
+  "rpc": [
+    {
+      "provider": "KiiChain",
+      "address": "https://rpc.kiivalidator.com"
+    }
+  ],
+  "snapshot_provider": "",
+  "sdk_version": "0.53.4",
+  "coin_type": "60",
+  "min_tx_fee": "400000000",
```

We changed this to a bit higher
```suggestion
  "min_tx_fee": "600000000",
```